tests/unit/localService.test.js

```diff
@@ -127,15 +160,18 @@ describe('localService', function () {
           result.added.should.be.an.Array();
           result.added.length.should.equal(2);
 
-          result.unchanged.should.containEql(fileList[0]);
-          result.changed.should.containEql(fileList[1]);
-          result.changed.should.containEql(fileList[2]);
-          result.changed.should.containEql(fileList[3]);
-          result.changed.should.containEql(fileList[6]);
-          result.deleted.should.containEql(currentWorkspace.files[4].path);
-          result.deleted.should.containEql(currentWorkspace.files[5].path);
-          result.added.should.containEql(fileList[4]);
-          result.added.should.containEql(fileList[5]);
+          result.unchanged.should.containEql(fileList[0]); // api1.raml
+          result.unchanged.should.containEql(fileList[1]); // api2.raml
+          result.changed.should.containEql(fileList[2]); // api3.raml
+          result.changed.should.containEql(fileList[3]); // api4.raml
+          result.changed.should.containEql(fileList[4]); // api5.raml
+          result.changed.should.containEql(fileList[6]); // another.json
```

Comment:
objects 'behaviour' can be inferred from the array in which they are placed. The idea of the comments is to match array positions to file names, which is useful when making changes to the test data.